include/NovelRT/PluginManagement/TemporaryFnPtrs.h

```diff
@@ -15,6 +15,7 @@ namespace NovelRT::PluginManagement
 {
     [[nodiscard]] std::shared_ptr<IGraphicsPluginProvider> GetVulkanPluginProvider() noexcept;
     [[nodiscard]] std::shared_ptr<IWindowingPluginProvider> GetGlfwPluginProvider() noexcept;
```

Comment:
We should probably change the name of this to `GetGlfwWindowPluginProvider` given the name of the other GLFW method.

Reply:
Done